public/modules/custom/grants_applicant_info/src/Element/ApplicantInfoComposite.php

```diff
@@ -201,8 +201,20 @@ protected static function getPrivatePersonFormElements(AtvDocument $grantsProfil
       '#wrapper_attributes' => [
         'class' => ['grants-handler--prefilled-field'],
       ],
-      '#suffix' => '</div>',
     ];
+    if (array_key_exists('phone_number', $profileContent)) {
+      $elements['phone_number'] = [
+        '#type' => 'textfield',
+        '#title' => t('Phone number'),
+        '#readonly' => TRUE,
+        '#required' => TRUE,
+        '#value' => $profileContent["phone_number"],
+        '#wrapper_attributes' => [
+          'class' => ['grants-handler--prefilled-field'],
+        ],
+        '#suffix' => '</div>',
```

Comment:
Voiks tää numero olla joskus poissa / ja jos näin käy, niin räjähtääkö koko ulkoasu kun tää sulkeutuva divi jää pois?

Reply:
Yksityishenkilöllä numero on pakollinen, eli siellä ei pitäs tulla. Tätä samaa koodia käytetään alempana toimintaryhmän tiedoissa, mutta sinne se jollain maagisella tavalla (ainakin siis works on my machine TM) laittaa sulkudivin vaikka siellä ei oo tätä puhelinnumeroa.

Reply:
Hetkinen... mä kävin approvee tän, mut eihän toi ees toimi ilman develop mergeä eka.

Ja sit pitäskö tohon sulkudiviin ottaa kantaa jotenki?